### ISSUE TITLE
ack manual ACK COLORS: typing errors `ACK COLOR_LINENO` + `ACK COLOR_COLNO`

### ISSUE THREAD

**Issue by vgx** [beyondgrep/ack3#364]
In the ack manual
https://beyondgrep.com/documentation/ack-v3.6.0-man.html
section ACK COLORS
https://beyondgrep.com/documentation/ack-v3.6.0-man.html#ACK-COLORS
introduction, we see two typing errors:

`    line no.    --color-lineno      ACK COLOR_LINENO    bold yellow`
`    column no.  --color-colno       ACK COLOR_COLNO     bold yellow`

Should be `ACK_...` (with underscore, not blank).

**Comment by vgx** [comment]
This issue had better been placed at beyondgrep/website, I guess. Sorry.

**Comment by petdance** [comment]
No, this is fine. The website is just a reproduction of the manual you get in `ack --man`.  Is the error in `ack --man` as well?

**Comment by vgx** [comment]
My given perl installation here (not under my control) seems to be less optimal.  `ack --man` does not produce the expected output (instead outputs "usage" as with `--help`).

```
$ ack --version
ack v3.6.0 (standalone version)
Running under Perl v5.34.0 at /usr/bin/perl.exe

Copyright 2005-2022 Andy Lester.

This program is free software.  You may modify or distribute it
under the terms of the Artistic License v2.0.

$ ack --man
Can't locate Pod/Usage.pm in @INC (you may need to install the Pod::Usage module) (@INC contains: /usr/lib/perl5/site_perl /usr/share/perl5/site_perl /usr/lib/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib/perl5/core_perl /usr/share/perl5/core_perl) at u:/_/scripts/ack line 2976.
Usage: ack [OPTION]... PATTERN [FILES OR DIRECTORIES]

...
```

(I don't care, and have web-search-found to blame it on the given perl installation, as said.)

In the single-file source code `=pod` section the two underscores are missing. 

```
1118 =pod
...
1931     Aspect      Option              Env. variable       Default
1932     --------    -----------------   ------------------  ---------------
1933     filename    --color-filename    ACK_COLOR_FILENAME  black on_yellow
1934     match       --color-match       ACK_COLOR_MATCH     bold green
1935     line no.    --color-lineno      ACK COLOR_LINENO    bold yellow
1936     column no.  --color-colno       ACK COLOR_COLNO     bold yellow
```

So `ack --man` would most likely contain the error as well (if it has worked with a better perl installation), I guess.

**Comment by petdance** [comment]
Ooops, there we are.

It's fixed with commit 8c2b539.  Thanks.

**Comment by n1vux** [comment]
> (I don't care, and have web-search-found to blame it on the given perl installation, as said.)

ah yes, some distributions (e.g. Linux flavors) that include Perl choose to put some "core" modules in separate packages. 

If using one of those, if it were (hypothetically) under your control, installing the OS/distro's "ack" package instead of CPAN modules or the one-file-download would bring `Pod::Usage` in also. 
Since Perl _assumes_ `Pod::Usage`  is core, CPAN tools do _not_ check and encourage adding it.
(This is alas a long-standing gap between expectations of Perl team and the several Linux teams.)

**If** you (_again hypothetically_) **needed** it to work, you _could_ collect the missing, needed Perl modules from CPAN (where they have dual-life availability for both this eventuality and for folks wanting to test/use updated versions of certain core that won't come from OS distro for years) into a `${HOME}/lib/perl5/lib/` or whatever and include that in your shell env **`PERL5LIB`** path.  Unless useful functionality is blocked, as opposed to `--man`, this likely doesn't matter.

A couple of tools can make living in unprivileged userland more tolerable. [`local::lib`](https://metacpan.org/pod/local::lib) manages your $HOME/lib for Perl.  [`PerlBrew`](https://metacpan.org/dist/App-perlbrew/view/script/perlbrew) goes one further (if you have a C compiler included) and lets you build one or more versions of perl to be your account's perl.
